packages/web/src/hooks/common/use-broadcast-handler.ts

```diff
@@ -125,18 +131,51 @@ export function makeBroadcastStakingMessage(
   };
 }
 
+export function makeBroadcastUnStakingMessage(
+  type: TNoticeType,
+  data: {
+    tokenASymbol: string;
+    tokenBSymbol: string;
+    tokenAAmount: string;
+    tokenBAmount: string;
+  },
+  hash?: string,
+): INoticeContent {
+  function description() {
+    switch (type) {
+      case "pending":
+        return `Unstaking <span>${data.tokenAAmount}</span> <span>${data.tokenASymbol}</span> and <span>${data.tokenBAmount}</span> <span>${data.tokenBSymbol}</span>`;
+      case "success":
+        return `Unstaked <span>${data.tokenAAmount}</span> <span>${data.tokenASymbol}</span> and <span>${data.tokenBAmount}</span> <span>${data.tokenBSymbol}</span>`;
+      case "error":
+        return `Failed to unstake <span>${data.tokenAAmount}</span> <span>${data.tokenASymbol}</span> and <span>${data.tokenBAmount}</span> <span>${data.tokenBSymbol}</span>`;
+    }
```

Comment:
If you can afford it, you can also think of a template.

Reply:
can you tell me more, i think we just inner this html in div.

Reply:
I didn't explain myself well enough.

I also agree with your comment.

I was thinking that when creating notification messages, it would be nice to manage them in the form of templates.
Previously, they were simple text, so I didn't bother creating them, but I thought it might be hard to manage if they had a lot of tags or variables.

This is what I had in mind.

```
// Declare template code to be readable 
const NOTIFICATION_SWAP_MESSAGE_TEMPLATE = `
<div>
  Swapping 
  <span>{tokenA}</span> 
  <span>{tokenASymbol}</span> 
  and 
  <span>{tokenB}</span>
  <span>{tokenBSymbol}</span>
</div>
`;
```

Create a function that maps the template's alternate keys.
```
mapMessageByTemplate(template: string, data: object) => string;
```

Reply:
What do you think?

Reply:
good idea sir